src/engine-scripts/puppet/userMenuState.js

```diff
@@ -0,0 +1,5 @@
+const menuState = require( './menuState' );
+module.exports = async (page, hashtags) => {
+    const isClosed = !hashtags.includes( '#userMenu-open' );
+    await menuState(page, '#p-personal-checkbox', isClosed);
```

Comment:
This works great, the only thing left I'm noticing is that the user menu seems to have an opacity transition on it that can lead to flaky tests (see below). Can the test wait for the transition to end (either with a 100ms delay or listening to the `transitionEnd` event)?
<img width="1609" alt="Screen Shot 2022-04-14 at 12 11 35 PM" src="https://user-images.githubusercontent.com/3496279/163451856-ef3a15ed-df96-48e9-86ce-6e7c4332927f.png">